model/user.go

```diff
@@ -557,6 +557,24 @@ func (u *User) GetDisplayName(nameFormat string) string {
 	return displayName
 }
 
+func (u *User) GetDisplayNameWithPrefix(nameFormat, prefix string) string {
+	displayName := prefix + u.Username
+
+	if nameFormat == SHOW_NICKNAME_FULLNAME {
+		if len(u.Nickname) > 0 {
+			displayName = u.Nickname
+		} else if fullName := u.GetFullName(); len(fullName) > 0 {
+			displayName = fullName
+		}
+	} else if nameFormat == SHOW_FULLNAME {
+		if fullName := u.GetFullName(); len(fullName) > 0 {
+			displayName = fullName
+		}
+	}
```

Comment:
Could we move this above if block into it's own function that we can call in both `GetDisplayNameWithPrefix` and `GetDisplayNameWith` so we're not duplicating code?